### DISCUSSION TITLE
eth-rpc-adapter - max batch size for RPC request

### DISCUSSION THREAD

**Discussion by andreclaro** [AcalaNetwork/Acala#2775] [2. Q&A]
Hello,

What's the max recommended batch size for RPC request (`MAX_BATCH_SIZE`/  `--max-batch-size`)?

The default is 50 but I would like to know what's the recommend maximum value?

https://www.npmjs.com/package/@acala-network/eth-rpc-adapter

Cheers,

**Comment by xlc** [comment]
50 is a good default for most of the use cases. You may want to reduce it for a public RPC service, or increase it for an internal RPC service that you have some applications is sending big batch requests.
